<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Tresa" failures="3" tests="12" name="com.naveenautomation.tests.CartPageTest" time="109.737" errors="0" timestamp="2024-03-10T06:32:31 EDT" skipped="9">
  <testcase classname="com.naveenautomation.tests.CartPageTest" name="validateInvalidCouponCode" time="9.599">
    <failure message="expected [Success: You have added iPod Classic to your shopping cart!
×] but found [Success: You have added iPod Classic to your shopping cart!
×]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Success: You have added iPod Classic to your shopping cart!
×] but found [Success: You have added iPod Classic to your shopping cart!
×]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.naveenautomation.tests.CartPageTest.validateInvalidCouponCode(CartPageTest.java:36)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- validateInvalidCouponCode -->
  <system-out/>
  <testcase classname="com.naveenautomation.tests.CartPageTest" name="validateInvalidCouponCode" time="9.761">
    <skipped/>
  </testcase> <!-- validateInvalidCouponCode -->
  <system-out/>
  <testcase classname="com.naveenautomation.tests.CartPageTest" name="validateHeaderCheckOutBtn" time="7.740">
    <skipped/>
  </testcase> <!-- validateHeaderCheckOutBtn -->
  <system-out/>
  <testcase classname="com.naveenautomation.tests.CartPageTest" name="validateEstimateShippingAndTaxesInfo" time="10.200">
    <failure message="expected [Success: You have added iPod Classic to your shopping cart!
×] but found [Success: You have added iPod Classic to your shopping cart!
×]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Success: You have added iPod Classic to your shopping cart!
×] but found [Success: You have added iPod Classic to your shopping cart!
×]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.naveenautomation.tests.CartPageTest.validateEstimateShippingAndTaxesInfo(CartPageTest.java:55)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- validateEstimateShippingAndTaxesInfo -->
  <system-out/>
  <testcase classname="com.naveenautomation.tests.CartPageTest" name="validateEstimateShippingAndTaxesInfo" time="11.076">
    <skipped/>
  </testcase> <!-- validateEstimateShippingAndTaxesInfo -->
  <system-out/>
  <testcase classname="com.naveenautomation.tests.CartPageTest" name="validateHeaderCheckOutBtn" time="8.432">
    <skipped/>
  </testcase> <!-- validateHeaderCheckOutBtn -->
  <system-out/>
  <testcase classname="com.naveenautomation.tests.CartPageTest" name="validateEstimateShippingAndTaxesInfo" time="10.361">
    <skipped/>
  </testcase> <!-- validateEstimateShippingAndTaxesInfo -->
  <system-out/>
  <testcase classname="com.naveenautomation.tests.CartPageTest" name="validateInvalidCouponCode" time="9.092">
    <skipped/>
  </testcase> <!-- validateInvalidCouponCode -->
  <system-out/>
  <testcase classname="com.naveenautomation.tests.CartPageTest" name="validateInvalidCouponCode" time="9.497">
    <skipped/>
  </testcase> <!-- validateInvalidCouponCode -->
  <system-out/>
  <testcase classname="com.naveenautomation.tests.CartPageTest" name="validateHeaderCheckOutBtn" time="7.590">
    <skipped/>
  </testcase> <!-- validateHeaderCheckOutBtn -->
  <system-out/>
  <testcase classname="com.naveenautomation.tests.CartPageTest" name="validateHeaderCheckOutBtn" time="7.238">
    <failure message="expected [Shopping Cart  (18.53kg)] but found [Shopping Cart  (10.53kg)]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Shopping Cart  (18.53kg)] but found [Shopping Cart  (10.53kg)]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.naveenautomation.tests.CartPageTest.validateHeaderCheckOutBtn(CartPageTest.java:75)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- validateHeaderCheckOutBtn -->
  <system-out/>
  <testcase classname="com.naveenautomation.tests.CartPageTest" name="validateEstimateShippingAndTaxesInfo" time="9.151">
    <skipped/>
  </testcase> <!-- validateEstimateShippingAndTaxesInfo -->
  <system-out/>
</testsuite> <!-- com.naveenautomation.tests.CartPageTest -->
